hw06_pipeline_execution/pipeline.go

```diff
@@ -10,5 +12,34 @@ type Stage func(in In) (out Out)
 
 func ExecutePipeline(in In, done In, stages ...Stage) Out {
 	// Place your code here.
-	return nil
+	inn := make(chan interface{})
+	st := stages[0](inn)
+
+	go func() {
+		for _, stage := range stages[1:] {
+			st = stage(st)
```

Comment:
```suggestion
			inn = stage(inn)
```

Reply:
./pipeline.go:20:10: cannot use stage(inn) (value of type <-chan interface{}) as chan interface{} value in assignment